engine/src/main/java/org/hibernate/search/engine/backend/work/execution/spi/OperationSubmitter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.backend.work.execution.spi;
+
+import java.util.Queue;
+
+import org.hibernate.search.engine.backend.work.execution.impl.OperationSubmitterType;
+
+
+/**
+ * Interface defining how operation should be submitted to the queue.
+ */
+public interface OperationSubmitter {
+	OperationSubmitter DEFAULT = OperationSubmitterType.BLOCKING;
```

Comment:
I've started with this `DEFAULT` keeping the config property in mind - so that it would be easier to find where we've ended up using this default and should make use of the config property. But as the property is not happening ... Removed `DEFAULT` 😃